lib/workload/stateless/universal-event-archiver/function/achiver.py

```diff
@@ -0,0 +1,43 @@
+import json
+import boto3
+from datetime import datetime, timezone
+import os
+
+# Initialize S3 client
+s3 = boto3.client('s3')
+
+# The name of the bucket
+BUCKET_NAME = os.getenv('BUCKET_NAME')
+
+def handler(event, context):
+    
+    # Current timestamp
+    now = datetime.now(timezone.utc)
+    time_stamp = str(now.timestamp()) # for object name
+    time_stamp_details = now.strftime("%Y-%m-%d__%H-%M-%S") # for tagging
+    
+    # Extract the event title (type) from detail type
+    event_title = event.get('detail-type', 'undefinedEvent')
```

Comment:
AWS uses white space in their `detail-type` values, which is not that nice in file names. 
It should not break anything and we can prevent that in our internal events, but I'd probably "sanitise" the `event_title` before using it in the object key.

Reply:
In case of any issues happened when used in filenames or URL, function ```sanitize_string``` will be applied:
* Removes any leading and trailing whitespace from the input string to clean it up before processing.
* [^\w]+: Matches any sequence of characters that are not alphanumeric or underscores. This includes special characters and spaces. "_" replaces the matched sequences with an underscore.
* strip('_') strip any leading and trailing underscore.

Test case: ```sanitize_string("  test %01## 23%!~@#$%^&*(). case.  ")``` will produce ```test_01_23_case```